### PR TITLE
Stepper improvements

### PR DESCRIPTION
**Pull request by JoeyDjr**
This PR improve the Stepper component by :

-  changing the icons of the buttons of the stepper.
-  reducing the icon size
-  introducing a container for the content and the buttons and setting a background color
-  adapting the storybook doc

### CODE REVIEW

**Review by ErikssonJoakim — APPROVED** [review]
LGTM 👍

File: stories/molecules/components/stepper/Stepper.stories.mdx [hunk 1]
```diff
@@ -43,7 +45,8 @@ import { useState } from 'react'
 
 The `Stepper` allows a user to follow a progression through different ordered steps.
 Each step can be validated to move on to the next step. The last step also provides the possibility to perform an additional action upon validation. Once all steps are completed, the user can reset the stepper.  
-It is then up to the user to define the height of the stepper as well as the content styles of each step. The content of the stepper is scrollable.
+It is then up to the user to define the height of the stepper as well as the content styles of each step. The content of the stepper is scrollable.  
+It is up to the caller to define the size (height and width) of the stepper, for example by placing it in a `div` and defining its size.
```
Comment (lolottetheclash) [inline stories/molecules/components/stepper/Stepper.stories.mdx:49]: I think the explanation regarding the user's control is duplicated, no?

Comment (JoeyDjr) [reply stories/molecules/components/stepper/Stepper.stories.mdx:49]: Not duplicated but redundant, I'll fix it

**Review by fredericvilcot — APPROVED** [review]
Seems good!

**Review by lolottetheclash — APPROVED** [review]
LGTM !

**Pull request merged**